src/cpu/base.cc

```diff
@@ -909,19 +909,19 @@ BaseCPU::diffWithNEMU(ThreadID tid, InstSeqNum seq)
     auto gem5_pc = diffInfo.pc->instAddr();
     diffAllStates->gem5RegFile.pc = gem5_pc;
     auto nemu_pc = diffAllStates->diff.nemu_commit_inst_pc;
-    DPRINTF(Diff, "NEMU PC: %#10lx, GEM5 PC: %#10lx, inst: %s\n", nemu_pc,
-            gem5_pc,
+    diffMsg += csprintf("In CPU%d: NEMU PC: %#10lx, GEM5 PC: %#10lx, inst: %s\n", cpuId(),
```

Comment:
It is not recommended to do this, because diffwithNemu was very frequently called, string operations will be very time consuming

Reply:
I think it is fine if performance overhead is less than 5%